sap_business_document_processing/document_classification_client/dc_api_client.py

```diff
@@ -0,0 +1,455 @@
+                               log_msg_after=f'Successfully deleted the document {document_id} from the '
+                               f'dataset {dataset_id}')
+        return response.json()
+
+    def get_dataset_info(self, dataset_id):
+        """
+        Gets statistical information about a dataset with a given ID
+        :param dataset_id: The ID of the dataset
+        :return: Summary information about the dataset that includes the number of documents in different processing
+        stages
+        """
+        response = self.get(DATASET_ENDPOINT(dataset_id=dataset_id),
+                            log_msg_before=f'Getting information about the dataset {dataset_id}',
+                            log_msg_after=f'Successfully got the information about the dataset {dataset_id}')
+        return response.json()
+
+    def get_datasets_info(self):
+        """
+        Gets summary information about the existing datasets
+        :return: An array of datasets
+        """
+        response = self.get(DATASETS_ENDPOINT,
+                            log_msg_before='Getting information about datasets',
+                            log_msg_after='Successfully got the information about the datasets')
+        return response.json()[API_DATASETS_FIELD]
+
+    def get_dataset_documents_info(self, dataset_id, top=None, skip=None, count=None):
+        """
+        Gets the information about all the documents in a specific dataset
+        :param dataset_id: The ID of an existing dataset
+        :param top: Pagination: number of documents to be fetched in the current request
+        :param skip: Pagination: number of documents to skip for the current request
+        :param count: Flag to show count of number of documents in the dataset
+        :return: Object that contains an array of the documents
+        """
+        params = {}
+        if top is not None:
+            params[API_PAGINATION_TOP_PARAM] = top
+        if skip is not None:
+            params[API_PAGINATION_SKIP_PARAM] = skip
+        if count is not None:
+            params[API_PAGINATION_COUNT_PARAM] = count
+        response = self.get(DATASET_DOCUMENTS_ENDPOINT(dataset_id=dataset_id), params=params,
+                            log_msg_before=f'Getting information about the documents in the dataset {dataset_id}',
+                            log_msg_after=f'Successfully got the information about the documents in the '
+                            f'dataset {dataset_id}')
+        return response.json()
+
+    def get_classification_documents_info(self, model_name, model_version):
+        """
+        Gets the information about recently classified documents
+        :param model_name: The name of the model against which the documents were classified
+        :param model_version: The version of the model against which the documents were classified
+        :return: An array of document information, each document information includes its reference ID and the
```

Comment:
Please see the https://github.com/SAP/business-document-processing/pull/29#discussion_r613941446

Reply:
Has been added